env_manager/tests/test_manager.py

```diff
@@ -47,20 +67,33 @@ def test_manager_backends(
     assert initial_package in " ".join(initial_list)
 
     # Install a new package in the created environment
-    manager.install(packages=[installed_package])
+    install_result = manager.install(packages=installed_packages, force=True)
+    assert install_result[0]
 
     def package_installed():
         package_list = manager.list()
-        return installed_package in " ".join(package_list)
+        for package in installed_packages:
+            return package in " ".join(package_list)
 
     wait_until(package_installed)
 
-    if backend != "mamba":
-        # Uninstall the new package
-        manager.uninstall(packages=[installed_package], force=True)
+    # Uninstall the new package
+    uninstall_result = manager.uninstall(packages=installed_packages, force=True)
+    assert uninstall_result[0]
 
-        def package_uninstalled():
-            package_list = manager.list()
-            return installed_package not in " ".join(package_list)
+    def packages_uninstalled():
+        package_list = manager.list()
+        for package in installed_packages:
+            return package not in " ".join(package_list)
+
+    wait_until(packages_uninstalled)
+
+    # Try to install unexisting package
+    error_result, message = manager.install(packages=errored_packages, force=True)
+    assert not error_result
+    assert error_message in message
 
-        wait_until(package_uninstalled)
+    # Try to uninstall unexisting package
+    # error_result, message = manager.uninstall(packages=errored_packages, force=True)
+    # assert not error_result
+    # assert error_message in message
```

Comment:
Why is this commented?

Reply:
Forgot to change this and uncomment it 😅